test/keras/test_sequential.py

```diff
@@ -4,13 +4,13 @@
 import syft as sy
 from syft import dependency_check
 
-if dependency_check.keras_available:
+if dependency_check.tfe_available:  # pragma: no cover
     import tensorflow as tf
     import tf_encrypted as tfe
 
 
-@pytest.mark.skipif(not dependency_check.keras_available, reason="tf_encrypted not installed")
-def test_instantiate_tfe_layer():
+@pytest.mark.skipif(not dependency_check.tfe_available, reason="tf_encrypted not installed")
+def test_instantiate_tfe_layer():  # pragma: no cover
```

Comment:
Could you make an issue for having a better way to no cover these?

Reply:
#2532